src/gluonnlp/sequence_sampler.py

```diff
@@ -458,7 +458,10 @@ def __init__(self, beam_size: int,
                  max_length_a: int = 0,
                  max_length_b: int = 200,
                  min_length: int = 1,
-                 stochastic: bool = False):
+                 stochastic: bool = False,
+                 sampling: bool = False,
+                 sampling_topp: float = -1.0,
+                 sampling_topk: int = -1):
```

Comment:
Add some documents here to describe different flags and also cite the paper.

- sampling_topp
   [[ICLR2020] The Curious Case of Neural Text Degeneration](https://arxiv.org/pdf/1904.09751.pdf)
- sampling_topk
   [[ACL2018]Hierarchical Neural Story Generation](https://www.aclweb.org/anthology/P18-1082.pdf)
- stochastic
   [[ICML2019] Stochastic Beams and Where to Find Them](https://arxiv.org/pdf/1903.06059.pdf)